message.go

```diff
@@ -551,6 +551,10 @@ func (r *messageSetReaderV2) readMessage(min int64,
 
 	headers = make([]Header, headerCount)
 
+	if headerCount >= 0 {
+		headers = make([]Header, headerCount)
+	}
```

Comment:
Do we need to remove the `headers = make([]Header, headerCount)` line above?

Reply:
Ah, oops! Yes